egs2/librispeech/ssl1/local/data.sh

```diff
@@ -37,35 +38,18 @@ if [ -z "${LIBRISPEECH}" ]; then
 fi
 
 if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
-    if [ ! -e "${LIBRISPEECH}/LibriSpeech/LICENSE.TXT" ]; then
-        echo "stage 1: Data Download to ${LIBRISPEECH}"
-        for part in dev-clean test-clean dev-other test-other train-clean-100 train-clean-360 train-other-500; do
-                local/download_and_untar.sh ${LIBRISPEECH} ${data_url} ${part}
-        done
-    else
-        log "stage 1: ${LIBRISPEECH}/LibriSpeech/LICENSE.TXT is already existing. Skip data downloading"
-    fi
+    local/data_asr1.sh --stage 1 --stop-stage 1 --data_url "${data_url}"
```

Comment:
I changed the data.sh to reuse the stages from asr1. Similar changes were done for librispeech/ssl1/local/data.sh